src/app/styles/index.scss

```diff
@@ -15,7 +15,79 @@
 }
 
 @layer components {
-    .h6 {
-        @apply font-rubik font-light text-2xl
-     }
+.primary {
+    @apply font-rubik font-normal text-xl
+ }
+
+ .primary-bold {
+    @apply font-rubik font-semibold text-[22px]
+ }
+
+ .additional {
+    @apply font-rubik font-normal text-base
+ }
+
+ .h6 {
+    @apply font-rubik font-light text-2xl
+ }
+}
+
+
+$backgroundImageColored: "data:image/svg+xml;utf8,<svg width='24' height='24' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M12 2L14.2451 8.90983H21.5106L15.6327 13.1803L17.8779 20.0902L12 15.8197L6.12215 20.0902L8.36729 13.1803L2.48944 8.90983H9.75486L12 2Z' fill='%23FFD805' /></svg>";
+$backgroundImageEmpty: "data:image/svg+xml;utf8,<svg width='24' height='24' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M12 2L14.2451 8.90983H21.5106L15.6327 13.1803L17.8779 20.0902L12 15.8197L6.12215 20.0902L8.36729 13.1803L2.48944 8.90983H9.75486L12 2Z' fill='%23A3A3A3' /></svg>";
+
+.rating {
```

Comment:
All these styles (line 36 - line 93 in commit 7ea4319aaffcd9e924e8c0d49e6204719ae1e1d4) apply only for StarRate component. So, please move them into separate file with styles attached only to StarRate component.